### PR TITLE
4th iteration

### PR DESCRIPTION
**Pull request by Yiyis**
This PR implemented the 4th iteration design on all existing pages.

Left-to-dos:
- archive ID references for institution record
- admin related pages design (we don't have them right now)
- filters (not working properly)
- the new artwork page: what, why & how
- user profile pages
- signup steps
- "last updated by" (I left it blank on some pages)
- responsive layout
- accessibility

[Implementation of the 4th iteration](https://app.gitkraken.com/glo/card/c8b346d86afd4e179d914e9d6c5267f6)

### CODE REVIEW

File: MainContainer/client/assets/maincontainer-style.css [hunk 1]
```diff
@@ -367,6 +339,20 @@ a.active-yellow span {
   width: 20%;
   height: auto;
 }
+.intro-box {
```
Comment (Mordax) [inline MainContainer/client/assets/maincontainer-style.css:342]: Nitpick: This should have a space between.

File: MainContainer/client/assets/maincontainer-style.css [hunk 2]
```diff
@@ -367,6 +339,20 @@ a.active-yellow span {
...
+  width: 82%;
+  margin: 0 auto;
+  box-shadow: 0px 3px 6px #00000029;
```
Comment (Mordax) [inline MainContainer/client/assets/maincontainer-style.css:345]: Nitpick: Improper hex value, should be #000029

File: MainContainer/client/assets/maincontainer-style.css [hunk 3]
```diff
@@ -367,6 +339,20 @@ a.active-yellow span {
...
+}
+
+.join-btn {
+  border: 2px solid #343638;
+  color: #343638;
+  padding: 5px 30px;
+}
+.sign-in-btn {
```
Comment (Mordax) [inline MainContainer/client/assets/maincontainer-style.css:353]: Nitpick: Missing a space between these two.

File: MainContainer/client/assets/maincontainer-style.css [hunk 4]
```diff
@@ -493,8 +498,8 @@ h5.underline-brush {
   height: 40px;
 }
 
-h5.underline-brush :before,
-h5.underline-brush :after {
+h1.underline-brush :before,
+h1.underline-brush :after {
```
Comment (Mordax) [inline MainContainer/client/assets/maincontainer-style.css:502]: Nitpick: These should have double colons for pseudo elements.

File: MainContainer/client/assets/maincontainer-style.css [hunk 5]
```diff
@@ -506,30 +511,28 @@ h5.underline-brush :after {
   background-position: 100% 100%;
 }
 
-h5.underline-brush :before {
+h1.underline-brush :before {
```
Comment (Mordax) [inline MainContainer/client/assets/maincontainer-style.css:514]: Nit: This should have a double colon

File: MainContainer/client/assets/maincontainer-style.css [hunk 6]
```diff
@@ -864,11 +867,8 @@ ul.social-network li {
     height: 20vw;
   }
 }
-
+/*
```
Comment (Mordax) [inline MainContainer/client/assets/maincontainer-style.css:870]: Quick question - how come this is commented versus being removed?

File: MainContainer/client/assets/searchservice-style.css [hunk 7]
```diff
@@ -71,14 +78,21 @@
...
   box-shadow: none;
 }
 
 #line::placeholder {
-  color: white;
+  color: #208778;
+  font-family: "FuturaStdLight", sans-serif;
+  font-size: 0.8em;
 }
 #line {
```
Comment (Mordax) [inline MainContainer/client/assets/searchservice-style.css:97]: Nit: Should have a space before

File: MainContainer/client/assets/maincontainer-style.css (hunk 2 above)
Comment (Yiyis) [reply MainContainer/client/assets/maincontainer-style.css:345]: The hex value should be alright, the two more numbers at the end are used to define the alpha value

Comment (Mordax) [reply MainContainer/client/assets/maincontainer-style.css:346]: Whoops, I meant it should be `#000029 `, Github just switches it to an issue reference. Hex values should be 6 digits in total.

Comment (Yiyis) [reply MainContainer/client/assets/maincontainer-style.css:346]: According to [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value), hex with alpha value is using 8 digits while normal hex is using 6 digits. On the design, the box-shadow is a little bit translucent so I used the alpha value. Did Github issue reference only complain here? I also used the 8 digits hex on "image-tint"

Comment (Mordax) [reply MainContainer/client/assets/maincontainer-style.css:346]: Interesting, I had no idea that you could do that. Just read about it [here](https://alligator.io/css/hex-code-colors-alpha-values/#adding-an-alpha-value-to-your-hex-codes). 

I've only seen transparency adjusted this way with rgba and a 4th value:
```css
box-shadow: 10px 10px 10px rgba(0, 0, 0, 0.5);
```
Then using Hex with alpha value is alright - if it's repeated everywhere, we should probably eventually write it down in a style guide for the VPIA.

**Comment by Mordax** [comment]
This next batch of updates is looking good on my end.

File: MainContainer/config/db.js [hunk 8]
```diff
@@ -7,10 +7,10 @@
...
-const caminte = require("caminte");
-const Schema = caminte.Schema;
+var caminte = require("caminte");
+var Schema = caminte.Schema;
 
-const config = {
+var config = {
   driver: "mysql",
   host: "localhost",
   username: "wapataho_vpia",
```
Comment (Mordax) [inline MainContainer/config/db.js:16]: Why have these changed from `const` to `var`? Is it due to your branch being older than develop?

**Review by Mordax — CHANGES_REQUESTED** [review]

**Review by Mordax — APPROVED** [review]

**Pull request merged**